### PR TITLE
TEST/COMPAT: Fix wire compat test failures - v1.17.x

### PR DESCRIPTION
**Pull request by yosefe**
## Why
Fix wire compat CI failures on v1.17.x branch

## How
Backport #10113 to v1.17.x + minor whitespace fix

### CODE REVIEW

**Comment by yosefe** [comment]
@brminich can you pls review?

**Review by brminich — APPROVED** [review]

**Review by tvegas1 — APPROVED** [review]

**Comment by brminich** [comment]
merging to unblock CI (some other CI fixes are blocked by the failures fixed in this PR)

**Pull request merged**
